### ISSUE TITLE
Displacement of the sight which stutter when shooting with a weapon

### ISSUE THREAD

**Issue by HavannaFR** [Facepunch/garrysmod-issues#4945]
### Details
For a few days now many players on my server have reported to me that they have problems when shooting a weapon, after many attempts to resolve I have highlighted the fact that they are all using the chromium x64 version.
After switching back to the stable version, they no longer had a problem. My server is using the stable version.
I know M9K is a terribly bad addon but it is still strange that the problem only happens on the chromium x64 version :/.
Recording 1 : https://www.youtube.com/watch?v=QQaHwXUAIYE
Recording 2 : https://www.youtube.com/watch?v=ZKYJKD9H_N8
Recording 3 : https://www.youtube.com/watch?v=WWDeebGnRuM

### Steps to reproduce
- Use Chromium x64
- Use M9K
- Fire with a weapon

**Comment by robotboy655** [comment]
Cannot confirm.

**Comment by MARK2580** [comment]
You did not even provide a link to this weapon, and the videos look very strange, as if the user himself moves the camera on purpose.

**Comment by HavannaFR** [comment]
As I said it is not an isolated problem, several of my players pointed out the problem to me, the 3 videos were recorded by 3 different people, here is the link of the weapon: https://steamcommunity.com/sharedfiles/filedetails/?id=128089118

**Comment by robotboy655** [comment]
I tried that addon and there are no such problems in my tests.

**Comment by HavannaFR** [comment]
I will do more tests on my side, I will keep you informed

**Comment by HavannaFR** [comment]
I tried by deactivating all my addons and leaving only m9k, in singleplayer I have the impression to have really very slightly the problem, it is hardly perceiptible (maybe that's why you didn't notice anything), but by going to a private server (tickrates 16) the problem is clearly visible. I then updated the server to 64 and then 128 tickrates and there were hardly any problems.

The problem is: the lower the tickrate, the more there is a problem with lags or some sort of small rollback of the sight.

I really want to stress that this problem is very recent.

Version : x86-64 - Chromium + 64-bit binaries
Addons : Only M9K Assault Rifles (https://steamcommunity.com/sharedfiles/filedetails/?id=128089118)
Map : gm_flatgrass
Tickrates : 16
